sdks/java/extensions/sql/build.gradle

```diff
@@ -85,6 +85,7 @@ dependencies {
   shadow project(path: ":sdks:java:core", configuration: "shadow")
   shadow project(":sdks:java:extensions:join-library")
   shadow library.java.slf4j_api
+  shadow "com.fasterxml.jackson.core:jackson-core:2.9.9"
```

Comment:
you can use `library.java.jackson_core`. It's defined [here](https://github.com/apache/beam/blob/a71bfda77df36aa1531f01533c372233cfba0dd9/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L496)

Reply:
ah, and please squash the commits